rspack/react-storybook/package.json

```diff
@@ -10,23 +10,20 @@
   },
   "dependencies": {
     "@swc/helpers": "0.5.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@modern-js/storybook": "latest",
```

Comment:
Perfer using a semver such as `^2.48.0`, so this example will not be broken when modern.js release v3